src/main/java/com/twilio/oai/resolver/node/NodeParameterResolver.java

```diff
@@ -27,6 +27,12 @@ public CodegenParameter resolve(final CodegenParameter codegenParameter) {
             codegenParameter.vendorExtensions.put("x-transform", transform);
         }
 
+        // Feature to allow desired methods to accept TwiMl objects
+        if (codegenParameter.paramName.equalsIgnoreCase("twiml")){
```

Comment:
Would it be better to move the configuration for the `twiml` type to the type config file, similar to what's done for Java and C#?